test/Faker/Provider/en_PH/AddressTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Faker\Test\Provider\en_PH;
+
+use Faker\Generator;
+use Faker\Provider\en_PH\Address;
+use Faker\Provider\en_PH\Person;
```

Comment:
Interesting, I thought I removed it. O.o 

But apparently, I did not.

Sorry for inconvenience.